src/Sulu/Bundle/CategoryBundle/Category/KeywordManager.php

```diff
@@ -37,7 +37,7 @@ public function __construct(
     ) {
     }
 
-    public function save(KeywordInterface $keyword, CategoryInterface $category, $force = null)
+    public function save(KeywordInterface $keyword, CategoryInterface $category, $force = null, bool $isPersisted = false)
```

Comment:
@wachterjohannes remove isPersisted again (we use `isNew` from the entity directly)